frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -66,14 +67,23 @@ export enum ColumnKey {
   AvailabilityEdge = 'availability_edge',
   AvailabilityFirefox = 'availability_firefox',
   AvailabilitySafari = 'availability_safari',
+  AvailabilityChromeAndroid = 'availability_chrome_android',
+  AvailabilityFirefoxAndroid = 'availability_firefox_android',
+  AvailabilitySafariIos = 'availability_safari_ios',
   StableChrome = 'stable_chrome',
   StableEdge = 'stable_edge',
   StableFirefox = 'stable_firefox',
   StableSafari = 'stable_safari',
+  StableChromeAndroid = 'stable_chrome_android',
+  StableFirefoxAndroid = 'stable_firefox_android',
+  StableSafariIos = 'stable_safari_ios',
   ExpChrome = 'experimental_chrome',
   ExpEdge = 'experimental_edge',
   ExpFirefox = 'experimental_firefox',
   ExpSafari = 'experimental_safari',
+  ExpChromeAndroid = 'experimental_chrome_android',
+  ExpFirefoxAndroid = 'experimental_firefox_android',
+  ExpSafariIos = 'experimental_safari_ios',
```

Comment:
For these WPT columns to work on sort, you will need to add them to the openapi document: https://github.com/GoogleChrome/webstatus.dev/blob/b13707e03db34c79525b8f7a4ab350991bc52498/openapi/backend/openapi.yaml#L61-L76